app/src/main/res/values/dimens.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <dimen name="margin_main_buttons">25dp</dimen>
+    <dimen name="margin_25dp">25dp</dimen>
+    <dimen name="text_title">35sp</dimen>
```

Comment:
In both cases, not allignment to the mobile code style. pls rename it

Reply:
done